src/redux/books/booksSlice.js

```diff
@@ -1,41 +1,103 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-unused-vars */
 /* eslint-disable no-param-reassign */
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import restructureApiResponse from '../../utils/restructureApiResponse';
+
+const baseAPI = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/ATwGr9HsrW4O0MQXnRyQ/books';
+
+export const getBooks = createAsyncThunk(
+  'books/getBooks',
+  // eslint-disable-next-line consistent-return
```

Comment:
It is always a good practice not to disable the linters since they are there to help and guide you produce the best standard codes. Kindly remove this and all other linter disabling lines from your code and make sure you do not disable the linters in this and all subsequent projects :pray